### PR TITLE
Feature/shivi

### PR DESCRIPTION
**Pull request by Shiveshlineup**
1. Header is completed
2. Left Pane is completed
3. Ignore Right pane

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>Kaampe</title>
```
Comment (kaamrepo) [inline index.html:7]: add Capital P - KaamPe

File: src/components/Header.jsx [hunk 2]
```diff
@@ -1,18 +1,72 @@
...
-            <li className='mx-5'>Employers</li>
-            <li className='mx-5'>Profile</li>
-        </ul>
+    <nav className="bg-green-700 p-3">
+      <div className=" mx-auto flex items-center justify-between">
+      <div className="container mx-auto flex items-center justify-between">
+        <div className='flex'>
+          <span className='text-white'><Briefcase /></span>
+          <a href="#" className="text-white text-xl font-bold ml-4">
+            Kaampe
```
Comment (kaamrepo) [inline src/components/Header.jsx:31]: add text as KaamPe

**Review by kaamrepo — CHANGES_REQUESTED** [review]
update Brand name Kaampe to KaamPe

File: index.html (hunk 1 above)
Comment (Shiveshlineup) [reply index.html]: done

**Pull request closed without merging**